src/components/ConfirmModal/index.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {
+    TouchableOpacity, Text, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Header from '../Header';
+import Modal from '../Modal';
+import styles from '../../styles/styles';
+import CONST from '../../CONST';
+import withWindowDimensions from '../withWindowDimensions';
+
+const propTypes = {
+    /** Title of the modal */
+    title: PropTypes.string.isRequired,
+
+    /** A callback to call when the form has been submitted */
+    onConfirm: PropTypes.func.isRequired,
+
+    /** A callback to call when the form has been closed */
+    onCancel: PropTypes.func.isRequired,
+
+    /** If the screen is small */
+    isSmallScreenWidth: PropTypes.bool.isRequired,
+
+    /** Modal visibility */
+    isVisible: PropTypes.bool.isRequired,
+
+    /** Confirm button text */
+    confirmText: PropTypes.string,
+
+    /** Cancel button text */
+    cancelText: PropTypes.string,
+
+    /** Modal content text */
+    prompt: PropTypes.string,
+};
+
+const defaultProps = {
+    confirmText: 'Yes',
+    cancelText: 'No',
+    prompt: '',
+};
+
+const ConfirmModal = ({
+    title, onConfirm, onCancel, confirmText, cancelText, prompt, isSmallScreenWidth, isVisible,
```

Comment:
NAB I would prefer to just see `props` here rather than a destructuring with so many items.

Reply:
Just because looking at this component for the first time, I first took time to look through and understand the prop types at the top of the file, I don't need them laid out again here. Plus I think it's clearer to see `props.someVariable` to differentiate props vs local variables such as `modalType` in this case.

Reply:
Sure will change it to props.